### PR TITLE
added chatwoot with vanilla HTML

### PR DESCRIPTION
**Pull request by Dhravya**

### CODE REVIEW

File: Chatwoot/chatwoot-vanilla-html/index.html [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+        (function (d, t) {
+            var BASE_URL = "https://app.chatwoot.com";
+            var g = d.createElement(t), s = d.getElementsByTagName(t)[0];
+            g.src = BASE_URL + "/packs/js/sdk.js";
+            g.defer = true;
+            g.async = true;
+            s.parentNode.insertBefore(g, s);
+            g.onload = function () {
+                window.chatwootSDK.run({
+                    websiteToken: 'Q4ESVSHz267aQCty5nGWW37R',
```
Comment (pramit-marattha) [inline Chatwoot/chatwoot-vanilla-html/index.html]: Please make sure you either use an environment variable or simply hide your token.

Comment (siddharth2798) [reply Chatwoot/chatwoot-vanilla-html/index.html]: @Dhravya

**Comment by Dhravya** [comment]
Hi there, i've made the changes, can you please review?

**Review by siddharth2798 — APPROVED** [review]

**Comment by siddharth2798** [comment]
Thanks for this!

**Pull request merged**